doc/modules/ROOT/pages/ref.adoc

```diff
@@ -1,17 +1,39 @@
 [#reference]
 = Reference
 
-* xref:reference/unordered_map.adoc[unordered_map]
-* xref:reference/unordered_multimap.adoc[unordered_multimap]
-* xref:reference/unordered_set.adoc[unordered_set]
-* xref:reference/unordered_multiset.adoc[unordered_multiset]
-* xref:reference/hash_traits.adoc[hash_traits]
+* xref:reference/header_unordered_map_fwd.adoc[`<boost/unordered/unordered_map_fwd.hpp>` Synopsis]
```

Comment:
Try putting the code backticks on the outside of the `xref` and tell me if you like how it looks.

![image](https://github.com/user-attachments/assets/8f4e03ff-2c24-4a45-8339-e36b601db647)

In this image, the top two links are the updated version. The third one is the current look.

Reply:
I can solve this in a more satisfactory way, for that I'd need for you to add this definition to the source `site.css`:

```
.inlink-code {
    font-family: Roboto Mono,monospace;
    font-size: 95%;
    color: inherit;
}
```

Can you do that?

Reply:
Of course. You can do that locally yourself even.

All you gotta clone is the unordered-ui-bundle repo and then invoke `gulp bundle` in it. There's docs here: https://gitlab.com/antora/antora-ui-default#user-content-development-quickstart

Reply:
I managed to do it without chaning the CSS file: 191adf4a3028b88010e9c4e735380fc79bddaffe.